### PR TITLE
feat: Google Map supp

### PR DESCRIPTION
**Pull request by chungwwei**

### CODE REVIEW

File: web-editor/public/google_preview.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="https://api.mapbox.com/mapbox-gl-js/v1.12.0/mapbox-gl.js"></script>
+    <script src="https://unpkg.com/@mapbox/mapbox-sdk/umd/mapbox-sdk.min.js"></script>
+    <script src="https://unpkg.com/@rtirl/api@latest/lib/index.min.js"></script>
+    <link
+      href="https://api.mapbox.com/mapbox-gl-js/v1.12.0/mapbox-gl.css"
```
Comment (juaoose) [inline web-editor/public/google_preview.html:8]: Do we need mapbox gl for this overlay to work?

File: web-editor/src/component/GoogleMapContainer.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  height: '524px',
+  marginTop: "16px"
+};
+
+
+export const GoogleMapContainer = ({ mapStyle, apiKey }) => {
+
+  console.log("google map container render");
+  
+  const pullKey = "justANeededHardCodedKey";
```
Comment (juaoose) [inline web-editor/src/component/GoogleMapContainer.jsx:15]: Forgot to add pullKey as a parameter to this component?

**Review by juaoose — CHANGES_REQUESTED** [review]

File: web-editor/public/google_preview.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+          top: 50%;
+          left: 50%;
+          transform: translate(-6px, -6px);
+          box-shadow: 0 0 10px cyan;
+        "
+      ></div>
+    </div>
+    <script>
+      var params = new URLSearchParams(window.location.search);
+      if(params.get("fullscreen")) {
```
Comment (juaoose) [inline web-editor/public/google_preview.html:26]: Maybe we could remove params not being used

File: web-editor/src/component/GoogleMapContainer.jsx (hunk 2 above)
Comment (chungwwei) [reply web-editor/src/component/GoogleMapContainer.jsx:15]: it's just previewing, the pullkey is not necessary, so we can just have a hardcoded value.

Comment (juaoose) [reply web-editor/src/component/GoogleMapContainer.jsx:15]: Oh ok, but then I guess we could remove that parameter if we're not using it.

**Review by juaoose — CHANGES_REQUESTED** [review]

**Review by juaoose — APPROVED** [review]

**Pull request merged**